.github/workflows/rws-submissions-checks.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: pip install publicsuffix2
       - name: Content check
         id: check
-        run: python3 main/check_sites.py -i pull-request/first_party_sets.JSON --data_directory=main --with_diff > results.txt
+        run: python3 main/check_sites.py -i pull-request/related_website_sets.JSON --data_directory=main --with_diff > results.txt
```

Comment:
This means that the workflow will be broken for this PR only, but should work afterward, right?

Reply:
Yeah, the current check is failing because it uses the main's version of the submission checks and is failing to find `first-party-sets.json` in this branch. Once this is merged, it shouldn't be a problem though.

Reply:
Right. Since there's another open PR, should we update the workflow so that it can handle either filename? Or are we planning on just asking the PR author to rebase and try again?

Reply:
It might be best to ask for the rebase. If we update the workflow and accept PRs with `first-party-sets.JSON`, we'd end up with 2 submission lists by accepting the PR and would have to resolve that ourselves, right?